.github/workflows/macos.yml

```diff
@@ -0,0 +1,47 @@
+name: Build & test on macOS
+
+on: [push, pull_request]
```

Comment:
To allow this to work from forks this should include 'pull_request_target' probably https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/

Reply:
It appears that I had an error in the expression that checked`matrix.use_homebrew`.

`pull_request_target` will [run the workflows which are defined in the base commit](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target), i.e. the ones that are in the "master" branch in this case — which won't work since we don't have any in there yet.